include/zephyr/drivers/pm_cpu_ops.h

```diff
@@ -51,6 +51,15 @@ int pm_cpu_off(void);
  */
 int pm_cpu_on(unsigned long cpuid, uintptr_t entry_point);
 
+/**
+ * @brief Power down the system
+ *
+ * This call is used to power down the whole system
+ *
+ * @retval 0 on success, a negative errno otherwise
```

Comment:
does this function actually returns?

Reply:
nah, that's fine, maybe add in the comment that with a sane SPCI implementation this should not return but otherwise is fine.